README.md

```diff
@@ -640,11 +640,10 @@ In general, you should avoid `NSError` in Swift in favor of defining your own `E
 
 ## Access Modifiers
 
-Always specify access modifiers to top-level definitions. Do not specify them otherwise unless
-the modifier is needed:
+Specify access modifiers only when they are needed and required by the compiler. For types and functions with the access level of `internal`, do not explicitly specify the access modifier since all entities in Swift are `internal` by default.
```

Comment:
How about:

For types and functions with the access level of `internal` -> For `internal` types and functions